components/admin_console/group_settings/group_details/group_details.jsx

```diff
@@ -93,7 +93,10 @@ export default class GroupDetails extends React.PureComponent {
         for (const channel of channels) {
             promises.push(this.props.actions.link(this.props.groupID, channel.id, Groups.SYNCABLE_TYPE_CHANNEL, {auto_add: true}));
```

Comment:
Just as a (small) naming observation as I was trying to follow the logic, it would prob be more readable (at least to me ) to have in index.js:
getGroupSyncables: getGroupSyncables (instead of fetchGroupSyncables)
linkGroupSyncable (instead of link): linkGroupSyncable,
unlinkGroupSyncable(instead of unlink): unlinkGroupSyncable

"link" is general but fetch/get is specific (target GroupsSyncables)

Reply:
I have to agree with this. If we could change the `link` and `unlink` to their full names, it would make code more clear and understanding.